api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -70,6 +73,9 @@ message PluginConfig {
 
   // Filter/service configuration used to configure or reconfigure a plugin
   // (proxy_on_configuration).
```

Comment:
Let's use name `proxy_on_configure` instead of `proxy_on_configuration ` for consistency with https://github.com/proxy-wasm/spec/tree/master/abi-versions/vNEXT